DataScience/dashboard_utils.py

```diff
@@ -120,8 +122,20 @@ def create_stats(log_fp, d=None, predictions_files=None, is_summary=False, repor
             else:
                 name = pred_fp.split('.')[-2] # check that policy name is encoded in file_name
             if name:
-                pred[name] = [x.strip() for x in open(pred_fp) if x.strip()]
-                print('Loaded {} predictions from {}'.format(len(pred[name]),pred_fp))
+                if log_type == 'cb':
+                    pred[name] = [x.strip() for x in open(pred_fp) if x.strip()]
+                elif log_type == 'ccb':
+                    with open(pred_fp) as f:
+                        pred[name] = []
+                        slot = []
+                        for x in f.read().splitlines():
```

Comment:
.splitlines() here seems to be detrimental: by removing it you have same functionality but python is going through the lines one by one without allocating memory